lib/mappers/details.js

```diff
@@ -129,17 +129,24 @@ function buildHistogram (container) {
   };
 }
 
-function extractComments (comments) {
-  if (!comments) {
+// Google Play responds with the country us and lang en
+// at ds:17 a single string 'er', but for other languages and countries
+// it responds correctly
+function extractComments (comments, rawInput) {
+  const fallbackComments = (!R.is(Array, comments))
+    ? R.path(['ds:19', 0], rawInput)
```

Comment:
I wonder if we could incorporate this to the mappings somehow, I guess it's ok if this is a one-time thing for this specific field, but this could easily go out of sync next time mappings change

Reply:
How about [this](https://github.com/facundoolano/google-play-scraper/pull/412/commits/d83a3efdb9e3a2695ee63c8b9d44d3fccd69dced)?

Reply:
yeah that seems better, just add a comment explaining why that fallback is sometimes necessary (and what that 'er' vaue means)

Reply:
done